hera_mc/cm_health.py

```diff
@@ -22,7 +22,7 @@ def check_for_overlap(interval):
     """
     maxx = max([x for x in interval[0] + interval[1] if x is not None]) + 100
     for i, ival in enumerate(interval):
-        interval[i] = [x if x is not None else maxx for x in ival]
+        ival[1] = maxx if ival[1] is None else ival[1]
```

Comment:
this is changing the functionality of this code, not just renaming things. Is it doing what you want?

Reply:
I started looking at the issue to address a "duplication" issue, so to make sure I checked and changed this line.  it worked before but I always struggled with the logic and this is easier.